common-content/en/module/induction/accounts/index.md

```diff
@@ -71,14 +65,25 @@ Your cohort is your first network, and this is one of the most valuable things y
 
 ### Slack
 
-[Slack](https://slack.com/intl/en-gb) is our community space and how we communicate with each other. Download and install it now.
+[Slack](https://slack.com/intl/en-gb) is our community space and how we communicate with each other. Download and install it now. If you are working on a library computer, log in to the web version instead.
 
 - [ ] I have downloaded the [Slack](https://slack.com/intl/en-gb) app on my computer and phone.
 - [ ] I have logged in to the {{<our-name>}} workspace.
 - [ ] I have found and joined my class channels.
 - [ ] I have updated my profile with my picture, my professional email address, my GitHub username, my Linkedin profile and my class.
 - [ ] I have introduced myself on Slack.
```

Comment:
Can we add: I have familiarised myself with Slack: [Slack Quick Guide](https://slack.com/intl/en-gb/help/articles/360059928654-How-to-use-Slack--your-quick-start-guide) and [Slack CYF Essentials](https://www.youtube.com/watch?v=3hgAjDHPypU)